app/controllers/teachers_controller.rb

```diff
@@ -1,4 +1,5 @@
 class TeachersController < ApplicationController
+  skip_before_action :verify_authenticity_token
```

Comment:
Hmm, this shouldn't be necessary any more. I assume you added this before fixing the ujs import?

Reply:
There's a very narrow edgecase here I originally added this in for that's still present where exiting out of the modal, opening it again, and then pressing the deny button without entering in a reason gives ActionController::InvalidAuthenticityToken. Not entirely sure why this is happening but I'll remove this for now and figure out it later